cartofacade/capi.go

```diff
@@ -376,11 +376,18 @@ func getConfig(cfg CartoConfig) (C.viam_carto_config, error) {
 		return C.viam_carto_config{}, err
 	}
 
-	vcc.map_rate_sec = C.int(cfg.MapRateSecond)
-	vcc.data_dir = goStringToBstring(cfg.DataDir)
+	// Remove cloud_story_enabled, map_rate_sec, and data_dir from C++ code
+	// JIRA Ticket: RSDK-52334 https://viam.atlassian.net/browse/RSDK-5334
+	vcc.cloud_story_enabled = C.bool(true)
+	vcc.data_dir = goStringToBstring("/tmp/")
+	if cfg.EnableMapping {
+		vcc.map_rate_sec = C.int(9000)
+	} else {
+		vcc.map_rate_sec = C.int(0)
+	}
```

Comment:
Could you explain what this will do?

Reply:
This is an artifact of avoiding the removal of map_rate_sec and data_Dir from the C++ code. Since it still relies on a configured map_rate_sec, I am creating said parameter at the last possible moment based on its new definition.